### PR TITLE
Update `angular-html-parser` to v2.1.0

### PR DESCRIPTION
**Pull request by fisker**
## Description

Pass `angularHtmlParser` around, instead of making everything async.

## Checklist

- [ ] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory).
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [ ] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/main/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
     "src/index.cjs",
     "standalone.js",
     "bin/",
-    "dist/",
+    "dist*/",
```
Comment (thorn0) [inline tsconfig.json:47]: Is this intended?

**Comment by thorn0** [comment]
Why not just import it statically like e.g. remark is loaded in `parser-markdown.js`?

File: tsconfig.json (hunk 1 above)
Comment (fisker) [reply tsconfig.json:47]: Yes, when debugging build script, I save different `dist` directories, so I ignored `dist*` https://github.com/prettier/prettier/blob/92a6af22e72cf1d937492fecc591bb41edd16306/.gitignore#L14

**Review by thorn0 — APPROVED** [review]
Looks good. But it'd be good to investigate what's going on with those dynamic imports.

**Review by sosukesuzuki — APPROVED** [review]

**Pull request merged**